frontend/src/css/App.css

```diff
@@ -14,29 +14,56 @@ All component specific CSS lives in the React file with the component.
 /* Headers */
 h3 {
   color: #009fae;
+  padding-top: 12px;
 }
 
 /* h1: add green line underneath */
 h1.green-divider:after {
   content: " ";
   display: block;
-  border: 2px solid #c7e68c;
+  border-bottom: 2px solid #c7e68c;
+  padding-bottom: 12px;
 }
 
 h2 {
   color: #137991;
+  padding-top: 12px;
 }
 
 /* Buttons */
 .btn-primary {
   background-color: #00565e;
 }
 
+.btn-primary:hover {
+  background-color: #327d78;
+}
+
+.btn-primary:active {
+  background-color: #00565e !important;
```

Comment:
Normally we wouldn't use !important, but it looks like Aurora does too, so we have to.